### PR TITLE
Add missing GHA group for building `llvm-bitcode-linker`

### PR DESCRIPTION
**Pull request by Kobzol**
Found while investigating https://github.com/rust-lang/rust/issues/127869.

r? @onur-ozkan

### CODE REVIEW

File: src/bootstrap/src/core/build_steps/tool.rs [hunk 1]
```diff
@@ -847,6 +847,15 @@ impl Step for LlvmBitcodeLinker {
...
 
+        let _guard = builder.msg_tool(
+            Kind::Build,
+            Mode::ToolRustc,
+            bin_name,
+            self.compiler.stage,
+            &self.compiler.host,
+            &self.target,
+        );
+
```
Comment (onur-ozkan) [inline src/bootstrap/src/core/build_steps/tool.rs:858]: Can you move this right above `cargo.into_cmd().run(builder);`?

Comment (Kobzol) [reply src/bootstrap/src/core/build_steps/tool.rs]: Done.

**Comment by Kobzol** [comment]
I wonder if we should eventually do this automatically somehow, e.g. by returning something like `WithGroup<CargoCommand>` from `prepare_tool_cargo`, which could automatically open/close groups in constructor/`Drop`.

**Comment by onur-ozkan** [comment]
> I wonder if we should eventually do this automatically somehow, e.g. by returning something like `WithGroup<CargoCommand>` from `prepare_tool_cargo`, which could automatically open/close groups in constructor/`Drop`.

Good idea, we can probably do the similar thing in other modules (e.g., `check.rs`) too. Feel free to r=me if you don't want to handle them in this PR.

**Comment by Kobzol** [comment]
Yeah this will require some experimentation, I don't want to do that in this PR.

@bors r=onur-ozkan rollup

**Pull request merged**